packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -26,18 +26,40 @@ class Config(OutputSchema.Config):
         }
 
 
+class GetProduct(OutputSchema):
+    name: ProductName
+    display_name: str
+    short_name: str | None = Field(
+        default=None, description="Short display name for SMS"
+    )
+
+    vendor: dict | None = Field(default=None, description="vendor attributes")
+    issues: list[dict] | None = Field(
+        default=None, description="Reference to issues tracker"
+    )
+    manuals: list[dict] | None = Field(default=None, description="List of manuals")
+    support: list[dict] | None = Field(
+        default=None, description="List of support resources"
+    )
+
+    login_settings: dict
+    max_open_studies_per_user: PositiveInt | None
+    is_payment_enabled: bool
+    credits_per_usd: NonNegativeDecimal | None
```

Comment:
Q1: who establishes this price? 
Q2: where is this Decimal value stored in? The DB? Does it have an equivalent type?

Reply:
A1: POs in the database
A2: yes. in the db using Numeric which is the equilvaent type